crypto/s2n_evp.h

```diff
@@ -27,7 +31,11 @@ struct s2n_evp_digest {
 
 struct s2n_evp_hmac_state {
     struct s2n_evp_digest evp_digest;
+#if defined(OPENSSL_IS_AWSLC) || defined(OPENSSL_IS_BORINGSSL)
+    HMAC_CTX *hmac_key;
+#else
     EVP_PKEY *mac_key;
+#endif
```

Comment:
I thought it might help readability:
* `HMAC_Init_ex(ws->p_hash.evp_hmac.hmac_key, ....)`
* `EVP_DigestSignInit(..., ws->p_hash.evp_hmac.mac_key)`

I agree naming it what they are improves this idea.